src/style.css

```
@@ -1,3 +1,86 @@
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --hover_color: #00acdc;
+  --background_color: #808080;
+  --header_background_color: hsl(0, 0%, 100%);
+  --text_color: hsl(0, 0%, 0%);
+  font-family: "Roboto", sans-serif;
+}
+
+html {
+  width: 100%;
+}
```

Move font-family to the html selector
```suggestion
}

html {
  font-family: "Roboto", sans-serif;
}
```